model/PerpendicularPaths.py

```diff
@@ -115,16 +116,17 @@ def display_update(self):
                 robot = self.robot_by_cell (point)
                 back_color = 0 if robot is None else robot.bgcolor()
                 if back_color == 0 and goal.point == point:
-                    back_color = 0x0050
+                    back_color = 0x0050 if os.name == 'nt' else 45
 
                 color = self.space_touched_by_xy (point)
-                windll.Kernel32.SetConsoleTextAttribute(std_output_hdl, color | back_color)
+                self.color_update (color, back_color)
                 #RODO: South wall
                 if c & self.directions[1].value == self.directions[1].value:
-                    print ("_", end="", flush=True)
+                    print ("_", end="")
                 else:
-                    print (".", end="", flush=True)
-                windll.Kernel32.SetConsoleTextAttribute(std_output_hdl, 15)
+                    print (".", end="")
```

Comment:
True flushes the buffer, by removing flush=true it made it work by using its false default.